### PR TITLE
feat:created login,logout and signup page

### PR DESCRIPTION
**Pull request by KamiyaGaikwad**

### CODE REVIEW

**Comment by KamiyaGaikwad** [comment]
SignuP page
image

**Comment by KamiyaGaikwad** [comment]
LoginIn Page
image

**Comment by KamiyaGaikwad** [comment]
LogOut
image

File: index.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          <div class="card-text">Check out our best Summer Collection to stay cool in style this season</div> -->
+          <!-- <div class="card-actions">
+          <a href="#" class="card-btn-1">READ</a>
+          <a href="#" class="card-btn-2">BOOKMARK</a>
+          <a href="#" class="card-icon-1"><i class="far fa-heart"></i></a>
+          <a href="#" class="card-icon-2"><i class="fas fa-share-alt"></i></a>
+          <a href="#" class="card-icon-3"><i class="fas fa-ellipsis-v"></i></a>
+        </div> -->
+        <!-- </div>
+        <img src="/Images/red-coat.jpg" class="card-img-1" alt="..."> -->
```
Comment (thegeorgenikhil) [inline index.html]: commented code

**Review by thegeorgenikhil — APPROVED** [review]
Good to merge!

**Review by Hnh013 — APPROVED** [review]
looks good to merge !

**Comment by Hnh013** [comment]
A suggestion : for future iterations , try to use either fixed or responsive units (px or rem)

File: index.html (hunk 1 above)
Comment (KamiyaGaikwad) [reply index.html]: Thank you for pointing it out. i will remove it

**Comment by KamiyaGaikwad** [comment]
Thank you for your suggestion @Hnh013 . i will keep that in mind next time

**Comment by Shah-Arsalan** [comment]
Give proper spacing to the navbar elements and remove color pallets from the logout page

**Comment by KamiyaGaikwad** [comment]
Thank you @Shah-Arsalan , I have done all the changes suggested by you
image
image
image

**Pull request merged**